libcontainer/intelrdt/intelrdt.go

```diff
@@ -169,15 +209,26 @@ func NewManager(config *configs.Config, id string, path string) *Manager {
 // newManager is the same as NewManager, except it does not check if the feature
 // is actually available. Used by unit tests that mock intelrdt paths.
 func newManager(config *configs.Config, id string, path string) *Manager {
+	var monitoringGroup bool
+	if config.IntelRdt.L3CacheSchema != "" || config.IntelRdt.MemBwSchema != "" || config.IntelRdt.ClosID != "" {
+		monitoringGroup = false
+	} else if config.IntelRdt.EnableCMT || config.IntelRdt.EnableMBM {
+		monitoringGroup = true
```

Comment:
@Creatone am reading this right, that if `ClosID` or `L3CacheSchema` or `MemBwSchema` is set, the settings for `EnableCMT` and `EnableMBM` are silently ignored?

If yes, such configuration should probably be rejected during validation.